Gopkg.toml

```diff
@@ -20,6 +20,10 @@ required = ["github.com/golang/protobuf/protoc-gen-go"]
   name = "k8s.io/apimachinery"
   version = "kubernetes-1.9.1"
 
+[[constraint]]
```

Comment:
I'm sad that we have to pull this into all of our Go projects when it's only the CLI that needs it 😢

Reply:
Yeah, I looked at implementing this strictly in terms of `client-go`, but I'd prefer to leverage https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/proxy/proxy_server.go#L182.

The good news is this should mostly slow the build the first time around.